### PR TITLE
SalesWings - Partner Docs

### PR DESCRIPTION
**Pull request by denis-egorushkin-sw**
# Pull Request/Issue Resolution

#### Description of Change:

The change adds the documentation for the SalesWings partnership with Braze.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [x] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [x] Check that all links work.
- [x] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [x] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [x] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by KellieHawks** [comment]
@denis-egorushkin-sw 
Please tag me (@KellieHawks) once the missing components have been added, and then the Braze documentation team can review it. Thanks!

**Comment by denis-egorushkin-sw** [comment]
@cla-bot check

**Comment by denis-egorushkin-sw** [comment]
@cla-bot check

**Comment by denis-egorushkin-sw** [comment]
Hi @KellieHawks and @alex-braze, this PR is now ready for review. Thank you for your support and looking forward to your feedback.

File: _docs/_hidden/wip_partnerships/saleswings/saleswings.md [hunk 1]
```diff
@@ -134,14 +147,14 @@ In order to make a URL trackable:
...
+  - Example of final link: {% raw %}`https://www.your-website.com?braze_id={{${braze_id}}}`{% endraw %}
 
 Option 2: **br_user_id** parameter:
-  - `?br_user_id=` followed by the “Personalization” variable `{{${user_id}}}`
+  - `?br_user_id=` followed by the “Personalization” variable {% raw %}`{{${user_id}}}`{% endraw %}
   - The variable here is your customizable “external ID”
-  - Example of final link: `https://www.client-website.com?br_user_id={{${user_id}}}`
+  - Example of final link: {% raw %}`https://www.client-website.com?br_user_id={{${user_id}}}`{% endraw %}
 
 The **braze_id** personalization variable is set to a permanent identifier of the user generated by braze and is always available. The **user_id** personalization variable is set to the identifier of the user in your system and might be missing in certain scenarios (e.g., for anonymous users created by braze SDK). If both **braze_id** and **br_user_id** are used in a link, SalesWings will consider only **braze_id** parameter.
```
Comment (alex-braze) [inline _docs/_hidden/wip_partnerships/saleswings/saleswings.md:159]: There are certain scenarios where the braze_id would no longer be available. For example, if the anonymous user was orphaned, the braze_id for the anonymous user would also be orphaned ([as shown in this doc](https://www.braze.com/docs/user_guide/data_and_analytics/user_data_collection/user_profile_lifecycle/#identified-user-profiles)) For this reason we may want to remove the term "permanent" when referencing the braze_id in this sentence - The **braze_id** personalization variable is set to a permanent identifier of the user generated by braze and is always available.

**Review by alex-braze — CHANGES_REQUESTED** [review]
@denis-egorushkin-sw just a couple of recommended edits:

1. Can we hold off on mentioning Currents until our teams have done some internal testing of this? We can easily add this to our docs in the near future when we are ready!
2. I added a comment in-line about braze_id.

**Review by alex-braze — APPROVED** [review]
Thanks for adessing the feedback!

**Comment by KellieHawks** [comment]
@denis-egorushkin-sw Closing this PR as your documentation has been addressed in a separate PR (https://github.com/braze-inc/braze-docs/pull/5470) as we ran into some Github permissioning issues during the review phase. Feel free to follow up with any questions!

-The Braze Documentation Team

**Pull request closed without merging**